packages/forklift-console-plugin/src/modules/Providers/views/details/components/CredentialsSection/components/list/OvirtCredentialsList.tsx

```diff
@@ -13,19 +20,31 @@ export const OvirtCredentialsList: React.FC<ListComponentProps> = ({ secret, rev
   const items = [];
 
   const fields = {
-    user: { label: t('Username'), description: t('RH Virtualization engine REST API user name.') },
+    user: {
+      label: t('Username'),
+      description: t(
+        'A user name for connecting to the Red Hat Virtualization Manager (RHVM) API endpoint. Ensure the user name is in the format of username@user-domain. For example: admin@internal.',
+      ),
+    },
     password: {
       label: t('Password'),
-      description: t('RH Virtualization engine REST API password credentials.'),
+      description: t(
+        'A user password for connecting to the Red Hat Virtualization Manager (RHVM) API endpoint.',
+      ),
     },
     insecureSkipVerify: {
       label: t('Skip certificate validation'),
-      description: t("If true, the provider's REST API TLS certificate won't be validated."),
+      description: t(
+        "If true (check box is checked), the provider's CA certificate won't be validated.",
+      ),
+      hint: t(
```

Comment:
hint is not specific to specific a field, hint is a form thing, please create a new const for the hint text, or just use it inline